xivo_dao/alchemy/endpoint_sip.py

```diff
@@ -410,3 +418,82 @@ def _find_first_value(self, section, key):
         matching_options = section.find(key)
         for _, value in matching_options:
             return value
+
+    @classmethod
+    def query_options_value(
+        cls,
+        option,
+        root_uuid=None,
+        sections=['endpoint'],
+        can_inherit=False
+    ):
+        check_sections = sections if isinstance(sections, list) else [sections]
```

Comment:
I don't see the point to search an option in many sections. There is no test and use case in the PR too